library-best-practices/JLBP-4.md

```diff
@@ -5,10 +5,10 @@
   public API within the same major version. For libraries following semantic
   versioning, this means libraries with a 0.x.y version. (See [JLBP-3](JLBP-3.md) 
   for more details on the recommendations for semantic versioning.)
-- Unstable features are not part of the public API of a
-  stable library. They are often marked with annotations like `@Beta`. (See
-  [JLBP-3](JLBP-3.md) for more details on the recommendations for annotating
-  unstable features.)
+- Features that are not part of the public API of a
+  stable library are called *unstable*. Unstable features in a stable library 
+  should be marked with annotations such as `@Beta`. (See
```

Comment:
Maybe we should make this more direct: "To mark a feature as unstable and not part of the public API, use annotations such as `@Beta`."

Reply:
Good idea. Done.